code/modules/psionics/faculties/coercion.dm

```diff
@@ -60,6 +60,19 @@
 		to_chat(user, SPAN_WARNING("\The [target] is in no state for a mind-read."))
 		return TRUE
 
+	var/mob/living/carbon/C = target
```

Comment:
This is a crash-possibility. We've never verified that the target is `carbon`.

So this and stuff up to line 72 need to be guarded with an `if (iscarbon(target))`.